tests/test_server.py

```diff
@@ -0,0 +1,366 @@
+import time
+import requests
+import httpx
+import os
+import asyncio
+import pytest
+import openai_responses
+from tempfile import NamedTemporaryFile
+import pandas as pd
+
+OPENAI_API_KEY = os.getenv("OPENAI_API_KEY")
+LS_API_KEY = os.getenv("LS_API_KEY")
+
+payload = {
```

Comment:
mind naming this something more descriptive than just "payload"? Also, would be helpful to have it in all caps so it's clear when it's used that this comes from the global namespace

Reply:
done